### ISSUE TITLE
Notification & links

### ISSUE THREAD

**Issue by csagan5** [Magisk-Modules-Repo/bromitewebview#1]
Hi there,

I find it a bit uneasy about a few things:
* in the README there is a link to the issue tracker of Bromite, but not any link to Bromite website; this means users can and will get to Bromite for support, but will have to type the link for donations; it's a dark pattern
* you have not mentioned in the README that Bromite developers (mostly me) are not involved, causing a sort of "implicit endorsing"; in these cases it's better to explicitly state that this is not an official project of Bromite
* I was not notified of this repository existence

I will wait for your feedback about this. Having said that, I have to say that there are two options going forward: you can keep this module as a separate effort as it is now, and hopefully you will address what mentioned above. Or we can look at making this module officially endorsed, somehow. In such case I would still want you to take care of this module for now and the future, but under the official Github Bromite project, under some conditions.

**Comment by alexa-v2** [comment]
I'll be happy to continue this discussion on Telegram

Tonight I'll edit the README as you asked but I've also included a license too

Also I'm not 100% sure of the donation link to be completely honest

**Comment by alexa-v2** [comment]
My Telegram username is @alexiacortez and the support group for this module @csagan5 is @inlmagisk

**Comment by alexa-v2** [comment]
@csagan5 edited README as requested

Would you care to elaborate on how I could get official?

**Comment by csagan5** [comment]
> edited README as requested. Is everything ok now?

Not really. The most important part is to tell people how to get the source code, that's also why I was asking to add proper hyperlinks. I will open a PR with my suggested changes.

> Would you care to elaborate on how I could get official?

We would have to put it under the Bromite Github organisation, and then Magisk-Modules-Repo forks it from there. At that point you can submit PRs to the upstream repo and we keep them in sync. I am however not sure about other changes needed to make it match the current project.

**Comment by alexa-v2** [comment]
PR #3 has been merged @csagan5

I would be happy to work with you and your team to bring this repo up to scratch. All I ask is credit given where it is due (don't take it over and claim all the credit).

**Comment by alexa-v2** [comment]
Also @csagan5 feel free to redirect users of my module to @inlmagisk for support, or my tracker at alexa-v2/bromitewebview

**Comment by csagan5** [comment]
> All I ask is credit given where it is due

All patches used in Bromite are correctly credited to original authors (to the best of my knowledge), and I ask the same for users of the Bromite patches (under GPLv3). The same should be done for this repository (respecting authorship and license).

> I would be happy to work with you and your team to bring this repo up
This is what I will do:
* create a `bromitewebview` repository under the Bromite organisation here on Github
* squash your commits and remove the binary blobs from them (so that the repository becomes smaller, it is currently ~384MB because of the binaries which were committed into it); the commits will preserve everything else (author, date)
* ask Magisk Modules admins to delete this repo and fork it from the Bromite official one
* ask Magisk Modules admins to invite us both to maintain the fork

This way we create the upstream repository in the Bromite organisation and the downstream repository under Magisk-Modules-Repo. You can also do the same for https://github.com/alexa-v2/bromite-systemless (fork from the new upstream repo) but I'd suggest you to rewrite the README to point at the new upstream and archive it, so that you maintain directly only the one under `Magisk-Modules-Repo`? It would be less work.

Going forward you can maintain directly the downstream one under the Magisk-Modules-Repo organisation and submit PRs to the upstream repo under the Bromite organisation, so we keep them in sync.

As for the support, I would suggest we use only the issue tracker on the official `bromitewebview` repository and you can keep using also @inlmagisk, since you will be the official maintainer of the Bromite SystemWebview Magisk Module.

**Comment by csagan5** [comment]
@alexa-v2 I have created the repository here: https://github.com/bromite/bromitewebview

There are two issues open there that need to be solved before it can be used; let's discuss them there.

**Comment by alexa-v2** [comment]
Both issue resolved I believe, and an issue has been opened to resubmit on the modules repository

**Comment by alexa-v2** [comment]
@csagan5 you should have wrote access to the repo

Push the changes from the bromite repo, I figured I'll wait until you do to update the the boot script to avoid conflicts

**Comment by csagan5** [comment]
> Both issue resolved I believe, and an issue has been opened to resubmit on the modules repository

The issues are not resolved; I would have preferred to proceed when the issues were resolved.
